flash/image/segmentation/input.py

```diff
@@ -124,20 +122,17 @@ def load_data(
         files = os.listdir(folder)
         files.sort()
         if mask_folder is not None:
-            mask_files = os.listdir(mask_folder)
-
-            all_files = set(files).intersection(set(mask_files))
-            if len(all_files) != len(files) or len(all_files) != len(mask_files):
-                rank_zero_warn(
-                    f"Found inconsistent files in input folder: {folder} and mask folder: {mask_folder}. Some files"
-                    " have been dropped.",
-                    UserWarning,
+            mask_files = {os.path.splitext(file)[0]: file for file in os.listdir(mask_folder)}
```

Comment:
```suggestion
            mask_files = {os.path.basename(file): file for file in os.listdir(mask_folder)}
```

Reply:
Can't switch for basename as they are not equivalent. Basename gives file name with extension but we need it without